package.json

```diff
@@ -10,21 +10,20 @@
     "start:dev": "./node_modules/nodemon/bin/nodemon.js --trace-warnings ./server.js --config nodemon.json",
     "start": "node src/index.js",
     "migrate": "NODE_ENV=development sequelize db:migrate --config src/config/databases.js --migrations-path src/db/migrations",
-    "docker:workers": "NODE_ENV=development docker exec -it resonate-api node src/jobs/queue-worker.js run",
```

Comment:
It turns out that because these were being set _before_ the docker command, they were being ignored. So I removed them. 

The docker-compose file sets the NODE_ENV anyway!